pkg/amqp-bunny/AmqpContext.php

```diff
@@ -131,6 +131,9 @@ public function createProducer(): Producer
     {
         $producer = new AmqpProducer($this->getBunnyChannel(), $this);
         $producer->setDelayStrategy($this->delayStrategy);
+        if (isset($this->config['publisher_confirm_timeout'])) {
+            $producer->enableConfirmMode((float) $this->config['publisher_confirm_timeout']);
```

Comment:
There must be a separate option for triggering the mode globally.  Say, `rabbitmq_publisher_confirm`. There must be a default value (could be 3) for a timeout.